### PR TITLE
LPS-112026 Fix questions listed with the wrong topic and fix links in MyActivity

### PR DESCRIPTION
**Pull request by LuismiBarcos**

### CODE REVIEW

**Comment by LuismiBarcos** [comment]
@nhpatt

**Comment by nhpatt** [comment]
Can we do it in another way? To try to avoid one extra query in the navigation bar, that component gets called a lot.

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/NavigationBar.es.js [hunk 1]
```diff
@@ -36,6 +37,11 @@ export default withRouter(
 			if (sectionTitle) {
 				context.setSection(sectionTitle);
 			}
+			else {
+				getSections(context.siteKey).then((sections) =>
```
Comment (nhpatt) [inline modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/NavigationBar.es.js:41]: Can we avoid this some other way?

**Comment by LuismiBarcos** [comment]
To minimize the calls, the getSections function only will be called when the application is completely lost, that is when the section title does not appear in the URL and is not stored in the context. At this moment, this should happen only when the app is at a specific page, like My Activity, and the page is completely reloaded or is the first time that is load

**Comment by nhpatt** [comment]
ci:forward

**Comment by nhpatt** [comment]
Rebased and formatted here: https://github.com/liferay-headless/liferay-portal/pull/21

**Pull request closed without merging**